_posts/2024-03-06-happy-birthday.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Happy birthday OpenTTD!
+author: orudge
+---
+
+Exactly 21 years ago, I received an ICQ message (look it up, kids) from a guy named Ludvig Strigeus (nicknamed Ludde). "Hello, you probably don't know me,
```

Comment:
Would you mind formatting this like all our post. For markdown we put a single sentence on a single line. So enter after dot.

Makes reviews easier, as do changes. And avoids the arbitrary location people hit enter :)

Tnx!

Reply:
Yes, noticed that after I pushed, sorry. Have adjusted now.